src/ride/ride.c

```diff
@@ -156,9 +156,29 @@ static void ride_update_vehicle_colours(int rideIndex);
 static void ride_set_vehicle_colours_to_random_preset(rct_ride *ride, uint8 preset_index);
 static void maze_entrance_hedge_removal(int x, int y, rct_map_element *mapElement);
 
+rct_ride *get_ride(int index)
+{
+	if (index < 0 || index >= MAX_RIDES)
+	{
+		log_error("invalid index %d for ride", index);
+		return NULL;
+	}
+	return &g_ride_list[index];
+}
+
+rct_ride_type *get_ride_entry(int index)
+{
+	return gRideTypeList[index];
```

Comment:
For rides there are, but I haven't seen any for `rct_ride_type`, only hint is https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/ride/ride.h#L440-L533